Hello/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>typer</title>
+	<meta charset="utf-8" />
+	<link rel="stylesheet" type="text/css" href="app.css">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
+	<script src="app.ts"></script>
```

Comment:
well it did work, because your app.ts does not contain any typescript code, just javascript, once you add typescript stuff, like typing, and ES6 features that are not natively supported by the browser you'll have some travel.
So the idea here is to write your `ts` file, transpile it using the command `tsc app.ts` this will generate a js file, browser compatible, and you can include that one.
But make sure to ignore it while commiting, add it to the `.gitignore` file